master/reference/felix/configuration.md

```diff
@@ -51,7 +52,10 @@ The full list of parameters which can be set is as follows.
 | `UsageReportingEnabled`           | `FELIX_USAGEREPORTINGENABLED`           | Reports anonymous {{site.prodname}} version number and cluster size to projectcalico.org. Logs warnings returned by the usage server. For example, if a significant security vulnerability has been discovered in the version of {{site.prodname}} being used. [Default: `true`] | boolean |
 | `UsageReportingInitialDelaySecs`  | `FELIX_USAGEREPORTINGINITIALDELAYSECS`  | Minimum delay before first usage report, in seconds. [Default: `300`] | int |
 | `UsageReportingIntervalSecs`      | `FELIX_USAGEREPORTINGINTERVALSECS`      | Interval at which to make usage reports, in seconds. [Default: `86400`] | int |
-
+| `VXLANEnabled`                    | `FELIX_VXLANENABLED`                    | Whether Felix should configure a VXLAN interface on the host. Set automatically to `true` by `{{site.nodecontainer}}` or `calicoctl` when you create a VXLAN-enabled pool. [Default: `false`] | boolean |
```

Comment:
Needs to be clearer that this should not be used by users (or else not documented here at all?)

(I thought, in error, that this was a way of disabling VXLAN use by calico to make it drop back to no encapsulation on pools).